app/serializers/trail_serializer.rb

```diff
@@ -0,0 +1,13 @@
+class TrailSerializer < ActiveModel::Serializer
+  attributes :id,
+            :name,
+            :city,
+            :state,
+            :country,
+            :lat,
+            :lng,
+            :description,
+            :directions,
+            :activities
```

Comment:
maybe we should align the parameters of a method call if they span more than one line.